app/views/thesis/process_theses.html.erb

```diff
@@ -19,80 +19,59 @@
   </div>
 </div>
 
-<% @theses.each do |thesis| %>
-  <div class="panel panel-<%= thesis.css_alert_type %>" data-id="thesis_<%= thesis.id %>">
-    <div class="panel-heading">
-      <div class="layout-3q1q layout-band">
-        <div class="col3q">
-          <%= thesis.user.name %>
+<% if show_form %>
+  <%= form_tag(process_path, {:method => :get}) do %>
+    <%= hidden_field_tag :status, request.params[:status]  %>
+    <%= hidden_field_tag :sort, request.params[:sort]  %>
+    <fieldset>
+      <legend>
+        Filter by graduation date
+      </legend>
+      <div class="group-inline">
+        Show theses from
+        <div class="field-wrap">
+          <%= label_tag(:start_month, "Start Month", { class: "field-label" }) %>
+          <%= select_tag(:start_month,
+                options_for_select([['', ''], ['January', '1'],
```

Comment:
We use simple_form on the user facing forms. It may simplify this stuff for us. Take a look if you haven't and see if it might be a good fit for what you are doing:
https://github.com/plataformatec/simple_form#available-input-types-and-defaults-for-each-column-type

Reply:
Recording our slack convo - it wasn't actually super obvious how to use simpleform when the form isn't bound to model instance, so let's not bother.